cli/sawtooth_cli/block.py

```diff
@@ -160,3 +153,6 @@ def get_block_data(block):
                 print_json(block)
             else:
                 raise CliException('unknown format: {}'.format(args.format))
+
+    else:
+        raise CliException('Unknown subcommand: {}'.format(args.subcommand))
```

Comment:
if these are specified via parsers, this should be an AssertionError, as argparse should catch incorrect commands